sdk/tests/deqp/data/gles3/shaders/linkage.test

```diff
@@ -3494,43 +3607,6 @@ group uniform "Uniform linkage"
 	end
 
 	group block "Uniform blocks"
-		case precision_mismatch
```

Comment:
Chrome still fails on this test. So there should be precision mismatch bugs for uniform blocks. Should we move it to conformance2 if it doesn't exist in native dEQP?